src/protocols/secure_channel/PASESession.h

```diff
@@ -340,13 +340,13 @@ class SecurePairingUsingTestSecret : public PairingSession
     SecurePairingUsingTestSecret()
     {
         SetLocalKeyId(0);
```

Comment:
Shouldn't we also rename this one to `SessionId` ?

Anyway, It is fine here, we can do it in following PRs

Reply:
Updated.